docs/testrunner/README.md

```diff
@@ -121,6 +121,7 @@ flavors:
         image:
           name: coreos
           version: latest
+        # architecture: amd64 # optional
```

Comment:
```suggestion
        architecture: amd64 # optional, for details see <link>
```
I'd remove the first comment char, it makes the correct indentation confusing and it now better aligns with the other optional fields example in this doc.
Can you maybe add a link to some doc on mcm/gardener side on potential values?

Reply:
There is no documentation available as of now. I have adapted the other change PTAL.